cypress/support/pages/articlePageObject.js

```diff
@@ -0,0 +1,87 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = '/#/editor';
+
+  get titleField() {
+    return cy.getByDataCy('article-title-field');
+  }
+
+  get descriptionField() {
+    return cy.getByDataCy('article-description-field');
+  }
+
+  get bodyArticleField() {
+    return cy.getByDataCy('article-body-textarea');
+  }
+
+  get tagField() {
+    return cy.getByDataCy('enter-tags-field');
+  }
+
+  get publishArticleBtn() {
+    return cy.getByDataCy('publish-article-btn');
+  }
+
+  get editArticleBtn() {
+    return cy.getByDataCy('edit-article-btn');
+  }
+
+  get deleteArticleBtn() {
+    return cy.getByDataCy('delete-article-btn');
+  }
+
+  get titleArticlePage() {
+    return cy.get('h1');
+  }
+
+  typeTitle(title) {
+    this.titleField.type(title);
+  }
+
+  typeDescription(description) {
+    this.descriptionField.type(description);
+  }
+
+  typeBodyArticle(body) {
+    this.bodyArticleField.type(body);
+  }
+
+  typeTag(tag) {
+    this.tagField.first().type(tag);
```

Comment:
The use of `.first()` in `typeTag` might not be necessary if the tag field is expected to be unique. Ensure that this is the intended behavior.